### DISCUSSION TITLE
How to change the url of the translation service correctly?

### DISCUSSION THREAD

**Discussion by Jaksor23** [YimMenu/YimMenu#1149] [Q&A] [answered]
That's right, even if I linked the master of the project I created on github based on [](https://github.com/YimMenu/Translations)

It always ends up pointing to the original YimMenu files and not to the ones in my github project, am I missing something maybe?
![Translation](https://user-images.githubusercontent.com/129014475/227812400-5f01e7ea-b3b4-4013-a6e1-2a09ca4cb8fe.png)

Thanks in advance

**Comment by Yimura** [comment, marked as answer]
Using jsdelivr for your translations requires you to wait 24h before changes will go in effect.

**Reply by Jaksor23** [reply]
Thanks, after waiting it worked perfectly!
